TestCases/parallel_regression.py

```diff
@@ -343,6 +343,20 @@ def main():
     contadj_incomp_cylinder.tol       = 0.00001
     test_list.append(contadj_incomp_cylinder)
 
+    ######################################                                                                                  
+    ### Harmonic Balance               ###                                                                                  
+    ######################################                                                                                    
+
+    spectral           = TestCase('spectral')
```

Comment:
Yes, I also changed the old SPECTRAL_METHOD option in the test case config files with the last commit.